data/agent/stagers/common/aes.py

```diff
@@ -3,6 +3,12 @@
 AES code from https://github.com/ricmoo/pyaes
 """
 
+from builtins import bytes
```

Comment:
The files in `stagers/common` are used as partials for the jinja2 templates which are payloads that get sent to an end user. Do we want to keep the stagers in a 2.7 state, is there harm in converting them? The main files that import these partials were left untouched by the migration tools because they are not syntactically correct python